reddit_liveupdate/pages.py

```diff
@@ -170,6 +171,24 @@ def __init__(self, **kwargs):
 
 
 class LiveUpdateEventFocusPage(LiveUpdateEventPage):
+    def __init__(self, focused_update, **kwargs):
+        og_data = {
+            "type": "article",
+            "url": make_event_url(c.liveupdate_event._id),
+            "description": trunc_string(
+                focused_update.body.strip(), MAX_DESCRIPTION_LENGTH),
+            "image": static("liveupdate-logo.png"),
+            "image:width": "300",
+            "image:height": "300",
+            "site_name": g.short_description,
```

Comment:
I think for open graph they really want a true site name here, not one with explainer text, so "reddit" would be best.